src/locales/en/translation.json

```diff
@@ -64,6 +64,7 @@
     "smartContract": "Smart Contract",
     "success": "Success",
     "status": "Status",
+    "test": "TEST",
```

Comment:
`common.test` path is so bad.

Reply:
My biggest question is, do we even want to use i18n for the tickers? It would be so easy if we could just use the name natively...

Reply:
I'm not sure if it needs to be translated :shrug:
For good i18n we would at least need `"{value} TEST"`, so languages with different order can swap `"TEST{value}"`.

Reply:
> I'm not sure if it needs to be translated 🤷🏻 

OK, asked on the slack channel, hopefully someone with a non-English background will chip in.

Reply:
Preliminary feedback seems to indicate that we don't need to translate the ticker symbols. In any case I have removed the dummy ROSE and TEST values from the translations.json file, and created a stub function for future extension if we want to translate them somehow.